### PR TITLE
Fix validate mainnet job

### PR DESCRIPTION
**Pull request by kderme**
https://github.com/input-output-hk/ouroboros-network/pull/2375 broke a nightly job that should be fixed.

### CODE REVIEW

File: ouroboros-consensus-byron/tools/db-converter/Main.hs [hunk 1]
```diff
@@ -48,22 +52,31 @@ deriving instance Show (Args Unwrapped)
...
     let epochFiles = filter (\f -> fileExtension f == ".epoch") files
     putStrLn $ "Writing to " ++ show dbDir
     for_ (sort epochFiles) $ \f -> do
       putStrLn $ "Converting file " ++ show f
-      convertChunkFile (EpochSlots epochSlots) f dbDir'
+      convertChunkFile (EpochSlots epochSlots) f dbDir' (mkProtocolMagicId protocolMagicId)
+
+-- | If `ProtocolMagicId` is not specified, we assume by default it's the
+-- mainnet.
+mkProtocolMagicId :: Maybe Word32 -> Word32
```
Comment (mrBliss) [inline ouroboros-consensus-byron/tools/db-converter/Main.hs:66]:
```suggestion
mkProtocolMagicId :: Maybe Word32 -> ProtocolMagicId
```

File: ouroboros-consensus-byron/tools/db-converter/Main.hs [hunk 2]
```diff
@@ -19,7 +19,8 @@ import           Control.Monad.Trans.Resource (runResourceT)
 import qualified Data.ByteString as BS
 import           Data.Foldable (for_)
 import           Data.List (sort)
-import           Data.Word (Word64)
+import           Data.Text (unpack)
```
Comment (mrBliss) [inline ouroboros-consensus-byron/tools/db-converter/Main.hs:22]:
```suggestion
import           Data.Text as T
```
There are so many `unpack`s (all `ByteString` types), so `T.unpack` is clearer.

File: ouroboros-consensus-byron/tools/db-converter/Main.hs [hunk 3]
```diff
@@ -48,22 +52,31 @@ deriving instance Show (Args Unwrapped)
...
     dbDir' <- parseAbsDir =<< canonicalizePath dbDir
     (_, files) <- listDir =<< parseAbsDir =<< canonicalizePath epochDir
     let epochFiles = filter (\f -> fileExtension f == ".epoch") files
     putStrLn $ "Writing to " ++ show dbDir
     for_ (sort epochFiles) $ \f -> do
       putStrLn $ "Converting file " ++ show f
-      convertChunkFile (EpochSlots epochSlots) f dbDir'
+      convertChunkFile (EpochSlots epochSlots) f dbDir' (mkProtocolMagicId protocolMagicId)
+
+-- | If `ProtocolMagicId` is not specified, we assume by default it's the
```
Comment (mrBliss) [inline ouroboros-consensus-byron/tools/db-converter/Main.hs:64]:
```suggestion
-- | If 'ProtocolMagicId' is not specified, we assume by default it's
```

**Comment by kderme** [comment]
bors merge

**Pull request merged**